### DISCUSSION TITLE
Explorer Content view - any way to remove the white lines?

### DISCUSSION THREAD

**Discussion by craciuncosmin** [ramensoftware/windhawk#301] [General]
This has been bugging me for a little bit, ever since I moved back to Windows after using Linux for some time. I am old and my eyesight is only getting worse. I enjoy the detail that Explorer's, well, 'Details' view can show to the user, but the text and icon size is too small on both my laptop and external monitors. I tried using the 'Content' view, given that it provides almost the same number of details, but I found myself dumbstruck by how distracting and ugly these white lines between items are:

![image](https://github.com/user-attachments/assets/5d946d21-c0b0-4d46-8c29-0e6f42f67bee)

I was wondering if there already is a Windhawk mod, or if a new one can be created, that would allow for the removal of these lines between items in Content view.

**Comment by m417z** [comment]
There's no existing mod that allows to do that. The folder view is a DirectUI control, and it should be possible to customize it by:
* Finding the relevant resource
* If it's stored in binary form, decompiling it with a tool such as [DuiTool](https://github.com/ahmed605/DuiTool)
* Modifying it as needed
* Using the [Resource Redirect](https://windhawk.net/mods/icon-resource-redirect) mod to make Explorer use the modified resource

So it's not something that can be done in five minutes, but that's the general direction. If you'd like to pursue it, you might want to join [the Windhawk Discord server](https://discord.com/servers/windhawk-923944342991818753), some people over there have more experience with DirectUI that I do.

**Reply by craciuncosmin** [reply]
thank you very much for the detailed explanation and clear direction. i'm going to start looking into this and hopefully be able to put something together

**Comment by Anixx** [comment]
Try this mod:
https://windhawk.net/mods/syslistview32-enabler
